src/index.css

```diff
@@ -1,13 +1,145 @@
+  --blue4: hsl(214, 65.8%, 17.9%);
+  --blue5: hsl(213, 71.2%, 20.2%);
+  --blue6: hsl(212, 77.4%, 23.1%);
+  --blue7: hsl(211, 85.1%, 27.4%);
+  --blue8: hsl(211, 89.7%, 34.1%);
+  --blue9: hsl(206, 100%, 50%);
+  --blue10: hsl(209, 100%, 60.6%);
+  --blue11: hsl(210, 100%, 66.1%);
+  --blue12: hsl(206, 98%, 95.8%);
+
+  /* Orange colors */
+
+  --orange1: hsl(30, 70%, 7.2%);
+  --orange2: hsl(28, 100%, 8.4%);
+  --orange3: hsl(26, 91.1%, 11.6%);
+  --orange4: hsl(25, 88.3%, 14.1%);
+  --orange5: hsl(24, 87.6%, 16.6%);
+  --orange6: hsl(24, 88.6%, 19.8%);
+  --orange7: hsl(24, 92.4%, 24%);
+  --orange8: hsl(25, 100%, 29%);
+  --orange9: hsl(24, 94%, 50%);
+  --orange10: hsl(24, 100%, 58.5%);
+  --orange11: hsl(24, 100%, 62.2%);
+  --orange12: hsl(24, 97%, 93.2%);
+
+  /* Gray Colors */
+
+  --gray1: hsl(0, 0%, 8.5%);
+  --gray2: hsl(0, 0%, 11%);
+  --gray3: hsl(0, 0%, 13.6%);
+  --gray4: hsl(0, 0%, 15.8%);
+  --gray5: hsl(0, 0%, 17.9%);
+  --gray6: hsl(0, 0%, 20.5%);
+  --gray7: hsl(0, 0%, 24.3%);
+  --gray8: hsl(0, 0%, 31.2%);
+  --gray9: hsl(0, 0%, 43.9%);
+  --gray10: hsl(0, 0%, 49.4%);
+  --gray11: hsl(0, 0%, 62.8%);
+  --gray12: hsl(0, 0%, 93%);
+
+  /* Black opacity */
+
+  --blackA1: hsla(0, 0%, 0%, 0.012);
+  --blackA2: hsla(0, 0%, 0%, 0.027);
+  --blackA3: hsla(0, 0%, 0%, 0.047);
+  --blackA4: hsla(0, 0%, 0%, 0.071);
+  --blackA5: hsla(0, 0%, 0%, 0.09);
+  --blackA6: hsla(0, 0%, 0%, 0.114);
+  --blackA7: hsla(0, 0%, 0%, 0.141);
+  --blackA8: hsla(0, 0%, 0%, 0.22);
+  --blackA9: hsla(0, 0%, 0%, 0.439);
+  --blackA10: hsla(0, 0%, 0%, 0.478);
+  --blackA11: hsla(0, 0%, 0%, 0.565);
+  --blackA12: hsla(0, 0%, 0%, 0.91);
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+img {
+  max-width: 100%;
+}
+
+input,
+button {
+  font-family: inherit;
+}
+
+button {
+  cursor: pointer;
+  font-size: 1rem;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-color: var(--gray1);
+  font-family: "IBM Plex Sans Arabic", sans-serif;
+  color: var(--gray12);
+}
+
+.p-x-1 {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.p-x-2 {
+  padding-left: 2rem;
+  padding-right: 2rem;
+}
+
+.p-y-2 {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+}
+
+.p-y-1 {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.m-y-1 {
+  margin: 1rem 0;
 }
+
+.container {
+  width: min(100%, 95em);
+  margin: 0 auto;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.main-container {
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  gap: 0.5rem;
+  min-height: 100vh;
+}
+
+/* @media screen and (min-width: 80em) {
```

Comment:
remove comments if not required

Reply:
Have updated code now. There were a couple of ideas of handling header and sidebar... will remove later.